integration-tests/src/test/resources/integration-test.anchor-config.yaml

```diff
@@ -8,7 +8,7 @@ stellar:
       settings: app-config                        # The location of the configuration data
     data-access:
       type: data-spring-jdbc                      # Activate [config-spring-jdbc] module.
-      settings: data-spring-jdbc-h2         # The location of the configuration data in this file.
+      settings: data-spring-jdbc-sqlite         # The location of the configuration data in this file.
```

Comment:
Same as above, h2 won't work because the stellar observer and the sep server need to share a common database, and H2 is not appropriate for that kind of setup.